contracts/SharesTimeLock.sol

```diff
@@ -228,6 +229,48 @@ contract SharesTimeLock is Ownable() {
     }
   }
 
+  /// @notice the migrator contract that will move veDOUGH -> veAUXO positions
```

Comment:
co-located state, events and methods for easier reviewing, these will be changed if we move ahead with implementation